sympy/series/limits.py

```diff
@@ -168,6 +168,16 @@ def doit(self, **hints):
         if e.is_Order:
             return Order(limit(e.expr, z, z0), *e.args[1:])
 
+        #Check to see if e is a simple power series, like (-1/2)**x
+        if e.is_Pow and len(e.free_symbols) == 1:
+            k = e.free_symbols.pop()
+            if k == z:
+                try:
+                    if (abs(e.base) < 1 and z0 is S.Infinity) or (abs(e.base) > 1 and z0 is S.NegativeInfinity):
```

Comment:
I think it's possible for it to still happen that `abs(e.base) < 1` when `e.base` is symbolic. It's better to check this explicitly (`not e.base.has(z)`).